qiskit/qobj/converters/lo_config.py

```diff
@@ -73,49 +77,80 @@ def __call__(self, user_lo_config):
         return self.qobj_model(**lo_config)
 
     def get_qubit_los(self, user_lo_config):
-        """Embed default qubit LO frequencies from backend and format them to list object.
-        If configured lo frequency is the same as default, this method returns `None`.
+        """Set expt level qubit LO frequencies. Use default values from job level if expt level
+        values not supplied. If expt level and job level values not supplied, raise an error. If
+        configured lo frequency is the same as default, this method returns ``None``.
 
         Args:
             user_lo_config (LoConfig): A dictionary of LOs to format.
 
         Returns:
-            list: A list of qubit LOs.
+            List[float]: A list of qubit LOs.
 
         Raises:
-            PulseError: when LO frequencies are missing.
+            QiskitError: When LO frequencies are missing and no default is set at job level.
         """
-        _q_los = self.qubit_lo_freq.copy()
-
-        for channel, lo_freq in user_lo_config.qubit_los.items():
-            self.default_lo_config.check_lo(channel, lo_freq)
-            _q_los[channel.index] = lo_freq
-
-        if _q_los == self.qubit_lo_freq:
-            return None
+        _q_los = None
+
+        # try to use job level default values
+        if self.qubit_lo_freq:
+            _q_los = self.qubit_lo_freq.copy()
+        # otherwise initialize list w ``None`` entries
```

Comment:
```suggestion
        # otherwise initialize list with ``None`` entries
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/705ca3c97ae84c6f950f430f1eea576d4eb93e1c.